main/proxy.go

```diff
@@ -17,9 +17,20 @@
 package main
 
 import (
-	"fmt"
+	"flag"
+	"io/ioutil"
+	"log"
+
+	"github.com/sciabarracom/incubator-openwhisk-runtime-go/openwhisk"
```

Comment:
you can update this now to `apache`?

Reply:
Yes and I recommend it, but It is an egg-and-chicken problem: I need to commit this PR before this code can work, and it will fail tests if I do BEFORE the commit so it may never be committed :)

Reply:
Will fix in the next PR once the code is available online to run my tests.